components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -180,6 +180,21 @@
  */
 @property(nonatomic, strong, nullable) UIFont *buttonFont;
 
+/**
+ If true, converts button titles to uppercase. Defaults to MDCButton's default (YES).
+ */
+@property(nonatomic) BOOL uppercaseButtonTitles;
```

Comment:
Nip naming: maybe `uppercaseButtonTitle` to align with the other two property names?